Ellen Homework 1.py

```diff
@@ -0,0 +1,31 @@
+import time 
+import random 
+
+
+random_creatures = ["gorgon", "vampire", "gruffalo", "zombie", "minatour"]
+
+
+def greet_user(username):
+    print("Hello, " + username + "!")
+    

```

Comment:
This is great! You can do the same with the intro just by taking all the print statements & conditionals we used in the last session and putting it in a function - I've done an example here: https://gist.github.com/arcadecoder/4486c0500fe288619fdaa1bf0d069111 

Then you can call the function using `intro()`